src/test/regress/expected/failure_savepoints.out

```diff
@@ -66,16 +66,26 @@ SELECT citus.mitmproxy('conn.onQuery(query="^RELEASE").kill()');
 (1 row)
 
 BEGIN;
-INSERT INTO artists VALUES (5, 'Asher Lev');
+UPDATE artists SET name='a';
 SAVEPOINT s1;
 DELETE FROM artists WHERE id=4;
 RELEASE SAVEPOINT s1;
+WARNING:  AbortSubTransaction while in COMMIT state
+WARNING:  could not issue cancel request
+DETAIL:  Client error: PQcancel() -- no cancel object supplied
```

Comment:
Not caused by this PR, but this is the first time I'm seeing this warning. Maybe we can suppress it in `SendCancelationRequest`?:
```c
    PGcancel *cancelObject = PQgetCancel(connection->pgConn);
    if (cancelObject == NULL)
    {
        /* connection failed, nothing to cancel */
        return false;
    }
```

Reply:
done